libqtile/widget/wlan.py

```diff
@@ -82,8 +104,10 @@ def poll(self):
         try:
             essid, quality = get_status(self.interface)
             disconnected = essid is None
+            ipaddr = get_private_ip(self.interface)
             if disconnected:
                 if self.use_ethernet:
+                    ipaddr = get_private_ip(self.ethernet_interface)
```

Comment:
I don't think we need to get the ip addr is we're not connected to wifi.

```suggestion
            ipaddr = ""
            if not disconnected:
                ipaddr = get_private_ip(self.interface)
            else:
                if self.use_ethernet:
                    ipaddr = get_private_ip(self.ethernet_interface)
```

Reply:
Thanks, added your suggestions, except I did
```
ipaddr = "N/A"
```
instead of 
```
ipaddr = ""
```
to be consistent.